.circleci/config.yml

```diff
@@ -62,31 +61,18 @@ jobs:
           command: npm run lint
       - run:
           name: Run Tests
-          command: npm t
-
-  lint-and-test-next-lts:
-    <<: *lint-and-test
-    executor: node-container-next-lts
-
-  test-built-app: &test-built-app
-    executor: node-container
-    steps:
-      - checkout
+          command: npm test
       - run:
-          name: Run Tests
+          name: Build app and run its tests
           environment:
-            MODE: 'ci'
+            MODE: 'local'
           command: |
             PROJECT=test-$(date +%s)
-            mkdir "${TMPDIR}/${PROJECT}"
-            cd "${TMPDIR}/${PROJECT}"
-            npx @contentful/create-contentful-app init test
-            cd test
-            npm t
-  
-  test-built-app-next-lts:
-    <<: *test-built-app
-    executor: node-container-next-lts
+            node lib/index.js init $PROJECT
+            cd $PROJECT
+            npm test
+            cd ..
+            rm -r $PROJECT
```

Comment:
If the test fails, you would have the workspace littered. This is why the temp folder was used before.

Also, we want to use "what we ship", rather than "what we think we ship" hence why we were using the `npx` command there

Reply:
Okay, I wasn't aware of that behavior of `npx`. From simply reading the code it looks like we would always run use the release version.